spec.html

```diff
@@ -4197,6 +4200,9 @@ <h1>ToBoolean ( _argument_ )</h1>
             </td>
             <td>
               Return *true*.
+              <emu-note>
+                <p>Alternate algorithm related to [[IsHTMLDDA]] Internal Slot can be found in <emu-xref href="#sec-IsHTMLDDA-internal-slot-to-boolean"></emu-xref>.</p>
```

Comment:
I would phrase it like this:

```suggestion
                <p>An alternate algorithm related to the [[IsHTMLDDA]] internal slot is mandated in section <emu-xref href="#sec-IsHTMLDDA-internal-slot-to-boolean"></emu-xref>.</p>
```